index.html

```diff
@@ -259,19 +259,41 @@ <h3>Language</h3>
             <div class="list-icon"></div>
             <h3>Frameworks</h3>
           </div>
-          
+
         </li>
 
         <li class="list">
           <div class="list-title">
             <div class="list-icon" id="circle"></div>
             <h3>Skills</h3>
           </div>
-         
+
         </li>
       </ul>
-    
+      <section class="form-section" >
+        <h1> Contact me</h1>
+        <p> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+        <form method="post" action="https://formspree.io/html/">
```

Comment:
- [ ] The form submission does not work, would you follow the get started button on [this link](https://formspree.io/html/) to learn how you should use formspree? , actually the `action` value should be similar to `https://formspree.io/f/{form_id}`

Reply:
Please , can you review my pull request ?

Reply:
i have created account on Formspree and update the action of my form